st2common/st2common/util/jinja.py

```diff
@@ -94,7 +95,10 @@ def get_filters():
         'use_none': use_none,
 
         'json_escape': json_escape.json_escape,
-        'jsonpath_query': jsonpath_query.jsonpath_query
+        'jsonpath_query': jsonpath_query.jsonpath_query,
+
+        'basename': path.basename,
+        'dirname': path.dirname
```

Comment:
Just a quick nit: Python ignores trailing commas in lists, tuples, and dictionary key/value...lists, so if we add a trailing comma here now, we don't have to see future diffs including silly things like:

```diff
-        'jsonpath_query': jsonpath_query.jsonpath_query
+        'jsonpath_query': jsonpath_query.jsonpath_query,
```

Reply:
Heh, yeah, just a personal preference here as well :)

I know there are two sides in Python (and more so in JavaScript for a different reason) - trailing comma and not, I'm personally on the "not" side :D